### PR TITLE
Adds Score's JSON Schema linter

### PR DESCRIPTION
**Pull request by rachfop**
Thank you for submitting this pull request! We appreciate you spending the time to work on these changes.

## What is the motivation?

docs-307

blocked:   https://github.com/score-spec/spec/pull/64
blocked: adds to schema store

## What does this change do?

Provide a description of what this pull request does.

## What is your testing strategy?

Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.

## Is this related to any issues?

If this pull request is related to any other pull request or issue, or resolves any issues - then link all related pull requests and issues here.

## Have you read the [Contributing Guidelines](CONTRIBUTING.md)?

- [ ] I have read the [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: content/en/docs/reference/ide-linter-autocomplete.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+weight: 10
+description: >
+  Reference documentation for the IDE linter and autocomplete for Score's JSON schema.
+---
+
+With the introduction of the JSON schema for Score, developers can now benefit from linter and autocomplete functionalities in their Integrated Development Environments (IDEs). IDEs like Visual Studio Code, JetBrains IDEs, and others support JSON schema validation and autocomplete out of the box or with extensions like [YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) for VS Code.
+
+## JSON schema
+
+Score's JSON Schema is supported by [JSON Schema Store](https://www.schemastore.org/json/); however, you can define the schema manually in your IDE.
```
Comment (sujaya-sys) [inline content/en/docs/reference/ide-linter-autocomplete.md:13]: Score is not yet supported by [JSON Schema Store](https://www.schemastore.org/json/) - will set up a separate ticket for that

**Comment by sujaya-sys** [comment]
@ghen @rachfop I suggest we publish this before adding it to the JSON schema store. We can add it later on

**Review by ghen — APPROVED** [review]

File: content/en/docs/reference/ide-linter-autocomplete.md (hunk 1 above)
Comment (sujaya-sys) [inline content/en/docs/reference/ide-linter-autocomplete.md:13]: Can we remove the part on [JSON Schema Store](https://www.schemastore.org/json/) until we have that in place?

File: content/en/docs/reference/ide-linter-autocomplete.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Generally, you need to add the following line to your IDE's settings:
+
+```json
+"json.schemas": [
+  {
+    "fileMatch": [
+      "score.yaml",
+      "score.yml"
+    ],
+    "url": "https://raw.githubusercontent.com/score-spec/score/main/schema/schema-v1b1.json"
```
Comment (sujaya-sys) [inline content/en/docs/reference/ide-linter-autocomplete.md:30]: This URL is now invalid. The schema can be found here: https://github.com/score-spec/schema

**Pull request merged**